### PR TITLE
Python3 doctest compatibility: algorithms beginning with F

### LINKED ISSUE #20726: Python3 doctest compatibility: algorithms starting with F
This issue is part of the move to Python3 in the doctests, 
Algorithms covered in this issue are:

    FFT
    FFTDerivative
    FFTSmooth
    FakeISISEventDAE
    FakeISISHistoDAE
    FakeMDEventData
    FilterBadPulses
    FilterByLogValue
    FilterByTime
    FilterByXValue
    FilterEvents
    FilterEventsByLogValuePreNexus
    FilterLogByTime
    FilterPeaks
    FindCenterOfMassPosition
    FindClusterFaces
    FindDeadDetectors
    FindDetectorsInShape
    FindDetectorsOutsideLimits
    FindDetectorsPar
    FindEPP
    FindPeakBackground
    FindPeaks
    FindPeaksMD
    FindReflectometryLines
    FindSXPeaks
    FindUBUsingFFT
    FindUBUsingIndexedPeaks
    FindUBUsingLatticeParameters
    FindUBUsingMinMaxD
    Fit
    FitGaussian
    FitPeak
    FitPowderDiffPeaks
    FitResolutionConvolvedModel
    FixGSASInstrumentFile
    FlatPlateAbsorption
    FlatPlatePaalmanPingsCorrection

### PR DESCRIPTION
**Pull request by josephframsay**
## Description of work.
This PR is part of refactoring effort to bring Python3 compatiblity into the doctests.
See https://github.com/mantidproject/mantid/issues/20672 for details.

The following tests were refactored to be compatible :
```
FFT
FFTDerivative
FFTSmooth
FakeISISEventDAE
FakeISISHistoDAE
FakeMDEventData
FilterBadPulses
FilterByLogValue
FilterByTime
FilterByXValue
FilterEvents
FilterEventsByLogValuePreNexus
FilterLogByTime
FilterPeaks
FindCenterOfMassPosition
FindClusterFaces
FindDeadDetectors
FindDetectorsInShape
FindDetectorsOutsideLimits
FindDetectorsPar
FindEPP
FindPeakBackground
FindPeaks
FindPeaksMD
FindReflectometryLines
FindSXPeaks
FindUBUsingFFT
FindUBUsingIndexedPeaks
FindUBUsingLatticeParameters
FindUBUsingMinMaxD
Fit
FitGaussian
FitPeak
FitPowderDiffPeaks
FitResolutionConvolvedModel
FixGSASInstrumentFile
FlatPlateAbsorption
FlatPlatePaalmanPingsCorrection
```
## To test.

Code review.

Fixes #20726 
Internal change, no release notes

### CODE REVIEW

File: docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst [hunk 1]
```diff
@@ -41,15 +41,15 @@ Usage
         ws.setY(i,np.array(deadDetArray))
         ws.setY(i+1,np.array(noisyDetArray))
 
-    print "With just the default LowThreshold of 0"
+    print("With just the default LowThreshold of 0")
     (wsOut,NumberOfFailures)=FindDetectorsOutsideLimits(ws)
-    print "%i spectra were outside the limits." % NumberOfFailures
-    print
+    print("{} spectra were outside the limits.".format(NumberOfFailures))
+    print("")
```
Comment (rosswhitfield) [inline docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst]: You could of just done `print()` here and one other place.

**Review by rosswhitfield — APPROVED** [review]
All `algorithms/F*` doc-tests now pass with python 3

File: docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst (hunk 1 above)
Comment (josephframsay) [reply docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst]: `print()` prints `()` in Python 2 - I kept it this way for backwards compatibility

Comment (rosswhitfield) [reply docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst]: 🤦‍♂️ 👍

**Pull request merged**